demo/demo-edge/hiboard-consumer/src/main/java/io/servicecomb/demo/edge/consumer/Consumer.java

```diff
@@ -55,17 +69,53 @@ public Consumer() {
   }
 
   public void run() {
+    prepareEdge();
+
+    invoke("/v1/add", 2, 1, addV1Result);
+    invoke("/v1/add", 3, 1, addV1Result);
+    invoke("/v1/add", 4, 1, addV1Result);
+    invoke("/v1/add", 5, 1, addV1Result);
+
+    invoke("/v1/dec", 2, 1, decV1Result);
+    invoke("/v1/dec", 3, 1, decV1Result);
+
+    invoke("/v2/add", 2, 1, addV2Result);
+    invoke("/v2/add", 3, 1, addV2Result);
+
+    invoke("/v2/dec", 2, 1, decV2Result);
+    invoke("/v2/dec", 3, 1, decV2Result);
+
+    printResults("v1/add", addV1Result);
```

Comment:
We should use assert instead of printResults to check the result data.

Reply:
done